### PR TITLE
organize the e2e case 

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve? 
organize the e2e case 

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

**Comment by yeya24** [comment]
Is it better to put different chaos in different files? Instead of a single `basic.go`

**Comment by Yisaer** [comment]
> Is it better to put different chaos in different files? Instead of a single `basic.go`

Currently, not necessary. I tend to make each file with a certain e2e environment.

File: test/e2e/chaos/basic.go [hunk 1]
```diff
@@ -59,581 +59,596 @@ var _ = ginkgo.Describe("[chaos-mesh] Basic", func() {
...
-				Selector: v1alpha1.SelectorSpec{
-					Namespaces: []string{
-						ns,
+	// pod chaos case in [PodChaos] context
+	ginkgo.Context("[PodChaos]", func() {
+
+		// podfailure chaos case in [PodFailure] context
+		ginkgo.Context("[PodFailure]", func() {
+
+			ginkgo.It("[Duration]", func() {
```
Comment (yeya24) [inline test/e2e/chaos/basic.go:68]: What's the difference between `Duration` case and `Schedule` case, seems they have almost the same test code, expect the chaos spec is a little different. Can we combine them together into one case?

**Pull request merged**